cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -112,6 +116,7 @@ public static KafkaConnectCluster fromConfigMap(ConfigMap cm) {
         if (kafkaConnect.isMetricsEnabled()) {
             kafkaConnect.setMetricsConfig(metricsConfig);
         }
+        kafkaConnect.parseAndSetLogging(Utils.getJson(data, KEY_CONNECT_LOG_CONFIG));
```

Comment:
Referring back to the `Logging` class comment, here you would have a `Utils.getLogging` which did the parsing for you.